<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>imageStackWidget</class>
 <widget class="QWidget" name="imageStackWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>120</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkViewAllImages">
        <property name="text">
         <string>View all images</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="layout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionClose_image">
   <property name="icon">
    <iconset resource="../resources/viewer.qrc">
     <normaloff>:/common/close.gif</normaloff>:/common/close.gif</iconset>
   </property>
   <property name="text">
    <string>Close image</string>
   </property>
  </action>
  <action name="actionDistribute_images">
   <property name="icon">
    <iconset resource="../resources/viewer.qrc">
     <normaloff>:/common/distribute.png</normaloff>:/common/distribute.png</iconset>
   </property>
   <property name="text">
    <string>Distribute images</string>
   </property>
  </action>
  <action name="actionClose_all_images">
   <property name="icon">
    <iconset resource="../resources/viewer.qrc">
     <normaloff>:/common/close.gif</normaloff>:/common/close.gif</iconset>
   </property>
   <property name="text">
    <string>Close all images</string>
   </property>
  </action>
  <action name="actionImage_type_stats">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Statistical image</string>
   </property>
  </action>
  <action name="actionStructural_image">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Structural image</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/viewer.qrc"/>
 </resources>
 <connections/>
</ui>
